contents/docs/tutorials/spa.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Tracking Single Page Apps
+sidebar: Docs
+showTitle: true
+---
+
+<br />
+
+<small class="note-block centered">_Estimated Reading Time: 8 minutes ☕☕_</small>
+
+<br />
+
+<span class="larger-image">
+
+![SPA Banner Image](../../images/tutorials/banners/spa.png)
+
+</span>
+
+<br />
+
+If you have a Single Page Application (SPA), and have been thinking about how to track it using PostHog - look no further.
```

Comment:
A small thing but I'd probably add a few examples of what SPAs are so that if a non-technical product manager is trying to use our docs to understand "will posthog work for us", they can answer their q.

Ie perhaps a few examples like 

"Single Page Applications are where the URL does not change when a user interacts. This is often applications written in:

* React
* etc
* Mobile apps (?)

"

Reply:
Hmm I wouldn't pinpoint a web framework here because it really isn't a feature of any of them. Rather I see a lot of Vanilla JS SPAs actually. Except maybe Angular is a tad bit more geared towards it. 

As for mobile apps, I can add a note about them

Reply:
Also, the URL may change, but the page won't refresh

Reply:
Also, the URL may change, but the page won't refresh